travis-bootstrap.sh

```diff
@@ -42,11 +42,11 @@ timed cabal update
 # issue, and it is easy enough to work around: clean first.
 
 echo Cabal
-(cd Cabal && timed cabal clean)
```

Comment:
Well, I have no idea why travis osx /bin/sh is /bin/bash, my osx has real `sh`, but I don't want to invest in this too much.